codalab/lib/download_manager.py

```diff
@@ -43,12 +43,20 @@ def __init__(self, bundle_model, worker_model, bundle_store):
     def get_target_info(self, uuid, path, depth):
         """
         Returns information about an individual target inside the bundle, or
-        None if the target doesn't exist.
+        None if the target or bundle doesn't exist.
 
         For information about the format of the return value, see
         worker.download_util.get_target_info.
         """
-        if self._bundle_model.get_bundle_state(uuid) != State.RUNNING:
+        try:
+            bundle_state = self._bundle_model.get_bundle_state(uuid)
+        except UsageError:
```

Comment:
`get_bundle_state(uuid)` should actually throw a `NotFoundError`, not a `UsageError`. This was our mistake -- if you could fix it, that would be great, otherwise I can make sure we fix it later ourselves. Thanks!